src/app/docs/angular-tree/angular-tree-docs.component.html

```diff
@@ -7,15 +7,110 @@
     Tree views provide hierarchical list views with multiple modes for selecting items in the lists.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-demo>
-  </sky-docs-demo> -->
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include toolbar"
```

Comment:
Should the *Include toolbar* option display a toolbar with dummy options? Looks kinda weird to have an empty toolbar.

Reply:
It should have expand/collapse controls at the very least. Does it not?
Also, if you turn on select mode, then the "select all" / "clear all" buttons should show up in the toolbar.

Reply:
I overlooked the expand/collapse controls because they were so far to the right when my browser was full screen. I was able to confirm the select all/clear all options as well. Thanks.